deployment/network-operator/charts/sriov-network-operator/templates/operator.yaml

```diff
@@ -88,6 +88,7 @@ spec:
             - name: RELEASE_VERSION
               value: {{ .Release.AppVersion }}
             - name: SRIOV_CNI_BIN_PATH
-              value: {{ .Values.operator.cniBinPath }}
+              # If global value is available, use that, otherwise, use the local one for the operator
```

Comment:
This chart should be kept aligned with upstream.
starting to keep changes locally is bad IMO.

the chart already defines cniBinPath

Reply:
that means -> end user is in charge of providing this path twice in the chart 
since helm does not support templating in values.yaml previous values.